src/lib.rs

```diff
@@ -44,7 +44,8 @@
 //!   `serde` crate.
 //!
 //! You need to enable one of the following Cargo features together with
-//! `v3`, `v4` or `v5` feature if you're targeting `wasm32` architecture:
+//! `v3`, `v4` or `v5` feature if you're targeting `wasm32-unknown-unknown`
+//! target:
 //!
 //! * `stdweb` - enables support for `OsRng` on `wasm32-unknown-unknown` via
```

Comment:
This isn't quite correct with `getrandom` anymore. platform support now follows [getrandom's](https://docs.rs/getrandom/0.2.0/getrandom/#webassembly-support). Both the `stdweb` and `wasm-bindgen` features now just enable `getrandom/js`, which works either with `wasm-bindgen` or `cargo-web`.